.github/CODEOWNERS

```diff
@@ -15,4 +15,10 @@
 # Global owners, will be the owners for everything in the repo. Membership is tracked via https://github.com/open-telemetry/community/blob/main/community-members.md 
 *   @open-telemetry/technical-committee @open-telemetry/specs-semconv-approvers
 
-# TODO - Add semconv area experts
\ No newline at end of file
+# Schemas and schema file tooling
+/schemas @tigrannajaryan
+/internal/tools/schema_check.sh @tigrannajaryan
+
+# Logs semantic conventions
+/semantic_conventions/logs @tigrannajaryan
+/specification/logs @tigrannajaryan
```

Comment:
Should we add @open-telemetry/specs-logs-approvers team as codeowner instead of listing individuals here?

Reply:
Do you think they are all semantic convention approvers for logs?

We can't provide any normative language in this specification that would alter the logging specification.

Reply:
Some of log spec approvers may be interested in semconv too, but I am not sure all well be.

@open-telemetry/specs-logs-approvers what do you think? If everyone wants to be a codeowner for log semantic conventions we can add the whole team, otherwise we can add only interested individuals.

Reply:
I can't commit to being an owner right now but feel free to ping me any time you want another perspective.